service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/Party.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.fpl.model.common;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import uk.gov.hmcts.reform.fpl.model.Address;
+
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+public class Party {
+    public final String partyID;
+    public final String partyType;
+    public final String firstName;
+    public final String lastName;
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+    public final Date dateOfBirth;
+    public final Address address;
+    public final EmailAddress email;
+    public final TelephoneNumber telephoneNumber;
```

Comment:
I suggest renaming `TelephoneNumber` type to `Telephone` to make it consistent with design. I will be easier to talk in the future.

Reply:
Sorry @swalker125 I meant renaming class / type not field. To match design we need:

```
public final Telephone telephoneNumber;
```

Reply:
<img width="900" alt="image" src="https://user-images.githubusercontent.com/1724691/61521178-94d52380-aa07-11e9-9496-547f2f3712f7.png">